src/ray/gcs/gcs_server/gcs_resource_manager.h

```diff
@@ -52,7 +52,9 @@ class GcsResourceManager : public rpc::NodeResourceInfoHandler {
   explicit GcsResourceManager(
       std::shared_ptr<gcs::GcsTableStorage> gcs_table_storage,
       ClusterResourceManager &cluster_resource_manager,
-      scheduling::NodeID local_node_id_ = scheduling::NodeID::Nil());
+      scheduling::NodeID local_node_id = scheduling::NodeID::Nil(),
```

Comment:
let's remove the default value `scheduling::NodeID local_node_id = scheduling::NodeID::Nil()`

to make things simple, we can always assume local_node_id will be passed into this class.